tomcat/src/main/java/org/apache/coyote/http11/HttpCookie.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.coyote.http11;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class HttpCookie {
+    private final Map<String, String> cookies;
+
+    private HttpCookie(final Map<String, String> cookies) {
+        this.cookies = cookies;
+    }
+
+    public static HttpCookie parseCookie(String cookie) {
+        Map<String,String> parsedCookie = new HashMap<>();
+        if(cookie!=null) {
+            parsedCookie = Arrays.stream(cookie.split(";= "))
```

Comment:
```suggestion
parsedCookie = Arrays.stream(cookie.split("; "))
```

오션은 ";= " 를 기준으로 나누었는데 어떤 기준인건지 궁금해요오~ 😁

Reply:
스플릿이 수정해주신대로 계획했었는데 밑에서도 '='을 사용하다보니 오타가 발생했던 것 같습니다 ㅠㅠ 수정했습니다!